bedrock/firefox/templates/firefox/os/ver/2.5.html

```diff
@@ -0,0 +1,109 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/os/index.html" %}
+
+{% block body_class %}{{ super() }} v2-5{% endblock %}
+
+{% block extrahead %}
+  {% stylesheet 'firefox_os_new' %}
+{% endblock %}
+
+{% block hero_copy %}
+  <div class="hero-content">
+    <h1>{{ _('Experience Firefox OS on your Android device')}}</h1>
+    <p>{{_('Introducing Firefox OS 2.5 Developer Preview, an experimental app that lets you to use Firefox OS on your Android device.')}}</p>
+    <div class="primary-cta-signup signup">
+      <a href="#TODO" class="cta-button" role="button">{{_('Get the Android App')}}</a>
+    </div>
+  </div>
+  <div class="hero-image">
+    {% block phone_image %}
+      {{ high_res_img('firefox/os-new/phone.png') }}
+    {% endblock %}
+  </div>
+{% endblock %}
+
+{% block features_headline %}
+  <h2>{{_('The Web is the platform')}}</h2>
+{% endblock %}
+
+{% block features_copy %}
+  <p>{{_('Firefox OS is the first truly open mobile platform built entirely on Web technologies. If you can build for the Web using HTML and CSS, you can build Firefox OS.')}}
+  <ul class="features">
+    <li class="customization">
+      <h3>{{_('The most customization')}}</h3>
+      <p>{{_('Make Firefox OS your own with apps and add-ons from the Firefox Marketplace, and keep all your favorite items front and centre with Pin the Web.')}}</p>
+    </li>
+    <li class="control">
+      <h3>{{_('The most control')}}</h3>
+      <p>{{_('From Do Not Track to Private Browsing with Tracking Protection, Firefox OS is designed keep you in control of your browsing activity.')}}</p>
+    </li>
+  </ul>
+{% endblock %}
+
+{% block fxox_apps %}
+{% endblock %}
+
+{% block fxox_news %}
+{% endblock %}
+
+{% block fxox_marketplace %}
+{% endblock %}
+
+{% block fxox_hub %}
+  <section class="fxos-hub">
+    <div class="content-wrapper">
+      <div class="hub-container">
+        <h2>{{_('Visit the Participation Hub')}}</h2> 
+        <p>{{_('Want to make Firefox OS even better? Get all the resources, tools and documentation you need from the Participation Hub.')}}</p>
+        <p><a class="more" href="#TODO">{{_('Check it out')}}</a></p>
```

Comment:
`#TODO` and also add `rel="external"` (assuming the link is external here too of course)